tools/Azure.Mcp.Tools.Redis/src/Commands/ResourceListCommand.cs

```diff
@@ -1,28 +1,28 @@
-// Copyright (c) Microsoft Corporation.
+﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
 using Azure.Mcp.Core.Commands;
 using Azure.Mcp.Core.Commands.Subscription;
-using Azure.Mcp.Tools.Redis.Models.CacheForRedis;
-using Azure.Mcp.Tools.Redis.Options.CacheForRedis;
+using Azure.Mcp.Tools.Redis.Models;
+using Azure.Mcp.Tools.Redis.Options;
 using Azure.Mcp.Tools.Redis.Services;
 using Microsoft.Extensions.Logging;
 
-namespace Azure.Mcp.Tools.Redis.Commands.CacheForRedis;
+namespace Azure.Mcp.Tools.Redis.Commands;
 
 /// <summary>
-/// Lists Azure Cache for Redis resources (Basic, Standard, and Premium tier caches) in the specified subscription.
+/// Lists Redis resources in a subscription. Returns Redis resource details for Azure Cache for Redis (Basic, Standard, and Premium tier), Azure Managed Redis, and Azure Redis Enterprise (`Balanced`, `MemoryOptimized`, `FlashOptimized`, `ComputeOptimized`, `Enterprise`, `EnterpriseFlash`).
 /// </summary>
-public sealed class CacheListCommand(ILogger<CacheListCommand> logger) : SubscriptionCommand<CacheListOptions>()
+public sealed class ResourceListCommand(ILogger<ResourceListCommand> logger) : SubscriptionCommand<ResourceListOptions>()
 {
-    private const string CommandTitle = "List Redis Caches";
-    private readonly ILogger<CacheListCommand> _logger = logger;
+    private const string CommandTitle = "List Redis Cache Resources";
+    private readonly ILogger<ResourceListCommand> _logger = logger;
 
     public override string Name => "list";
 
     public override string Description =>
         $"""
-        List/show all Redis Caches in a subscription. Returns Redis Cache details including Azure Cache for Redis resources (Basic, Standard, and Premium tier caches). Use this command to explore and view which Redis Cache resources are available in your subscription.
+        List/show all Redis Cache resources in a subscription. Returns Redis Cache details including Azure Cache for Redis resources (Basic, Standard, and Premium tier caches), Azure Managed Redis clusters, and Azure Redis Enterprise clusters. Use this command to explore and view which Redis Cache resources are available in your subscription.
```

Comment:
```suggestion
        List/show all Redis resources in a subscription. Returns details of all Azure Managed Redis, Azure Cache for Redis, and Azure Redis Enterprise resources. Use this command to explore and view which Redis resources are available in your subscription.
```
(removed "cache" and "cluster" terminology; removed tiers, listing AMR first)